lib/train/transports/ssh_connection.rb

```diff
@@ -52,6 +52,7 @@ def initialize(options)
       @bastion_host           = @options.delete(:bastion_host)
       @bastion_user           = @options.delete(:bastion_user)
       @bastion_port           = @options.delete(:bastion_port)
+      check_config_options
```

Comment:
I might suggest the name `read_options_from_ssh_config`. The word `check` implies that there is some validation going on.

Reply:
sure will do @clintoncwolfe, thanks